### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by juwara0**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [ ] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [X] #major# - incompatible API change

Closes: https://github.com/ciena-frost/ember-frost-popover/issues/61

# CHANGELOG
* **Added** `ember-frost-test` @ `^4.0.0`
* **Added** `chai-jquery` @ `^2.0.0`
* **Updated** `ember-cli-chai` to `0.4.3`
* **Updated** to pin `ember-cli-code-coverage` @ `0.3.12`
* **Updated** `ember-cli-mocha` to `0.14.4`
* **Added** `ember-hook` @ `1.4.2`
* **Added** `ember-sinon` @ `^0.7.0`
* **Updated** `ember-test-utils` to `^8.1.0`
* **Added** `sinon-chai` @ `^2.14.0`
* **Added** ignoring of `package-lock.json` until we are ready to move to node 8
* **Removed** useLintTree ember-cli-mocha configuration from `ember-cli-build.js`
* **Removed** blueprint file since dependent packages are now included via dependencies
* **Updated** `ember-cli-frost-blueprints` to `^5.0.1`
* **Removed** `ember-computed-decorators` and removed necessary babel config
* **Updated** `ember-concurrency` to be a dependency instead of devDependency
* **Removed** unused `ember-elsewhere` package
* **Updated** `ember-prop-types` to `^6.0.1` and moved to dependency instead of devDependency
* **Removed** unused `ember-spread` package
* **Removed** unused `ember-truth-helpers` package
* **Updated** `ember-cli-sass` to `7.1.1`
* **Updated** `ember-frost-core` to `^5.0.0`
* **Removed** previous `.eslintrc` file which has now been replaced by `.eslintrc.js`
* **Removed** `.remarkrc` file since it is now provided by `ember-test-utils`

### CODE REVIEW

**Comment by notmessenger** [comment]
Remove _/visual-acceptance_ from _.npmignore_

**Comment by notmessenger** [comment]
In _addon/components/frost-popover.js_ can you please remove line 87 and 128 and remove the `//` from line 474.

File: ember-cli-build.js [hunk 1]
```diff
@@ -3,12 +3,6 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon')
 
 module.exports = function (defaults) {
   let app = new EmberAddon(defaults, {
-    babel: {
-      optional: ['es7.decorators']
```
Comment (notmessenger) [inline ember-cli-build.js]: Don't still need this?  I have not been checking for this entry in the other repos we've been updating.

Comment (juwara0) [reply ember-cli-build.js]: It is needed for `ember-computed-decorators` which is not being used in this add-on.

**Comment by notmessenger** [comment]
👍

**Pull request merged**